test/api._coffee

```diff
@@ -120,7 +133,7 @@ expectDZI = (act, exp={}) ->
     for prop in ['width', 'height', 'tileSize', 'tileOverlap']
         _expectNumberWithin act[prop], 0, Infinity
 
-    _expectStringMatching act.tileFormat, /^(png|jpeg)$/
+    _expectStringMatching act.tileFormat, /^(png|jpg|jpeg)$/
```

Comment:
I forgot to ask here: @iangilman @gasi: is `jpg` a valid DZI tile format? Is `jpeg` *not*? I noticed that VIPS always generates `jpg`, but I thought I remembered DZIs back in the day being `jpeg`.